content/3_Silver/Binary_Search.mdx

```diff
@@ -319,20 +375,23 @@ public class BinarySearch {
 <PySection>
 
 ```python
-def firstTrue(lo, hi, f):
-	# returns smallest x in [lo,hi] that satisfies f
-	# hi+1 if no x satisfies f
-	hi+=1
+from typing import Callable
+
+
```

Comment:
```suggestion
```

Reply:
ryan
it's the convention to have 2 newlines between functions and anything so
i'm just gonna